test/unit/formats/errors.test.js

```diff
@@ -1,4 +1,6 @@
 /*
+      - main
+      - main
```

Comment:
Duplication needed? Not sure this commented-out content is intended.

Reply:
Completely accidental change. As of right now these workflows are unfinished but was hoping for input. Biggest hurdle is I don't have perms to add the necessary secrets to the repository to allow the workflows to trigger the workflow PR checks. 

Overall I think it'd be nice to have these introduced to allow us to manage/maintain releases from the repository directly. Main motivation here for me is to easier manage shipping 'pre' release variants for testing fixes with the broader community, along with standardizing the process for anyone and everyone.

Reply:
I also don't have those permissions, but generally favor good CI/CD practices.

Reply:
@DABH Is this something you could assist with?

Reply:
Hey @maverick1872 , great to hear from you, and thank you so much for the contributions!  Looks like some interesting work here.  Let me know what kind of secrets you need added and where -- I believe I should have permissions to add such things.  Feel free to email me if there is sensitive info we don't want sitting publicly in the repo.

Reply:
(And, one comment -- if this works for `winston`, and is the way we want to release things, then we'd probably want to set up this type of thing for `winston-transport` and `logform` as well, among other packages...)

Reply:
Sounds good. I'll replicate these over to a test repo to finalize all the requirements in terms of secrets, then I'll send the list your way. I should be able to setup the other repo such that y'all can test the process and give any feedback on it if you so desire.